types/lodash/index.d.ts

```diff
@@ -280,9 +280,9 @@ declare namespace _ {
         (value: number): LoDashImplicitWrapper<number>;
         (value: string): LoDashImplicitStringWrapper;
         (value: boolean): LoDashImplicitWrapper<boolean>;
-        (value: number[]): LoDashImplicitNumberArrayWrapper;
-        <T>(value: T[]): LoDashImplicitArrayWrapper<T>;
-        <T extends {}>(value: T): LoDashImplicitObjectWrapper<T>;
+        (value: number[] | null | undefined): LoDashImplicitNumberArrayWrapper;
+        <T>(value: T[] | null | undefined): LoDashImplicitArrayWrapper<T>;
+        <T extends {}>(value: T | null | undefined): LoDashImplicitObjectWrapper<T>;
```

Comment:
Actually I think I found a way to handle case 2 without bloating the types too badly.